variables.tf

```
@@ -24,18 +6,24 @@ variable "environment_variables" {
     "value" = "TRUE"
   }]
 
-  description = "A list of maps, that contain both the key 'name' and the key 'value' to be used as additional environment variables for the build."
+  description = "A list of maps, that contain both the key 'name' and the key 'value', and optionally 'type' (with the value 'PARAMETER_STORE') to be used as additional environment variables for the build."
+}
+
+variable "description" {
+  type        = "string"
+  default     = ""
+  description = "The AWS Codebuild project description. Generated based on the label module if left empty."
 }
 
 variable "enabled" {
   type        = "string"
-  default     = "true"
+  default     = true
```

```suggestion
  default     = "true"
```

Let's not change it here, it's not related to the PR description.

It also could break top-level modules that depend on string type.
Also, Terraform recommends using strings instead of booleans for all input variables because of Terraform handles booleans differently from TF files and from the command line.